backend/src/services/release.ts

```diff
@@ -248,7 +252,9 @@ export async function getModelReleases(
   const model = await getModelById(user, modelId)
 
   const auths = await authorisation.releases(user, model, results, ReleaseAction.View)
-  return results.filter((_, i) => auths[i].success)
+  return results
```

Comment:
If we care about performance here, we can combine this `filter` and `map` into a single `reduce`:
```
return results.reduce<ReleaseInterface>((updatedResults, result, index) => {
  if (auths[index].success) {
    updatedResults.push({ ...result, semver: semverObjectToString(result.semver) })
  }
  return updatedResults
}, [])
```

Reply:
Added, but wasn't sure on how to implement the updatedResults as I can't push to it as its not an array, etc